packages/stable/src/api/geospatial/types.ts

```diff
@@ -49,12 +49,12 @@ export interface FeaturesCreateItem {
     | { geojson: object };
 }
 
-export interface Features extends FeaturesCreateItem {
+export interface Feature extends FeatureCreateItem {
   createdTime: Date;
   lastUpdatedTime: Date;
 }
 
-export interface FeatureTypes {
+export interface FeatureType {
   id: CogniteInternalId;
   externalId: CogniteExternalId;
```

Comment:
What is CogniteExternalId and how it is different from ExternalId?

Reply:
CogniteExternalId just tells that id is a string whereas
ExternalId is { externalId : id } shaped object. Generally used in search.

Reply:
https://github.com/cognitedata/cognite-sdk-js/blob/708763f5568ad4dc4ddc48e3e2bedc2f2bc732c6/packages/core/src/types.ts#L37